packages/dbos-cloud/applications/delete-app.ts

```diff
@@ -1,5 +1,5 @@
 import axios, { AxiosError } from "axios";
-import { isCloudAPIErrorResponse, handleAPIErrors, getCloudCredentials, getLogger, retrieveApplicationName } from "../cloudutils";
+import { isCloudAPIErrorResponse, handleAPIErrors, getCloudCredentials, getLogger, retrieveApplicationName } from "../cloudutils.js";
```

Comment:
Come on, that file doesn't even exist :-P

Reply:
![titanic meme](https://y.yarn.co/1d1b72db-b232-404f-af27-aca7c9de80f2_text.gif)

Reply:
Doesn't exist, and will probably not ever exist (at that particular path relative to the .ts file).
It's weird that for all the ESM perfectionism and zealotry nobody bothered to fix this.  It's an artifact of the implementation, and tsc could easily fix it when it (or whatever else processes your ts) compiles.  This is sort of like including the .o file from your .c file.